src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,271 @@
+import { Component } from "react";
```

Comment:
Because of the double quotes, I know this is an auto import from vscode!  Let's switch it to single quotes for consistency.

Reply:
I changed all of the files so that they'd have double quotes based on your previous comment about JSX.

Reply:
That will work.  I typically do single quotes for everything outside of the return statement (where the JSX lives), but again, I don't care as long as we are consistent!